### PR TITLE
Creating the Fibonnaci program in C and a README file

### PR DESCRIPTION
**Pull request by rsm23**
Creating the Fibonnaci program in C and a README file

### CODE REVIEW

File: hello_world/hello_world.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
```
Comment (t2013anurag) [inline hello_world/hello_world.c]: you can remove this, as this is already implemented.

File: programs/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## The Fibonacci Sequence
```
Comment (t2013anurag) [inline programs/README.md]: create this file inside the fibonacci directory.

File: programs/fibonacci.c [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
```
Comment (t2013anurag) [inline programs/fibonacci.c]: directory name should be fibonacci

**Comment by rsm23** [comment]
Please review it now

**Comment by t2013anurag** [comment]
@rsm23 - hey, remove the programs directory, just add fibonacci directory.

**Comment by t2013anurag** [comment]
@rsm23 - https://github.com/hacktoberfest17/programming/tree/master/fib
See this, just add the code here. Also the existing readme is not good, you can replace it with yours.

Change the name of directory from fib to fibonacci too.

**Comment by rsm23** [comment]
@t2013anurag No prob bro, will do

**Pull request closed without merging**